fnConsensus/config.go

```diff
@@ -18,6 +18,11 @@ type OverrideValidator struct {
 	VotingPower int64
 }
 
+type FnConcensusSigner struct {
+	Enabled   bool
+	Validator bool
+}
```

Comment:
Maybe just add a new field to the existing reactor config, don't think we need a whole new sub-section in the config.